src/setup/templates/rgw.logrotate.tmpl

```diff
@@ -0,0 +1,13 @@
+TMP_LOG_PATH/*.log
```

Comment:
This is how default ceph logrotate looks like? If not, can you post here the default ones

Reply:
default ceph logrotate file:

> /var/log/ceph/*.log {
    rotate 7
    daily
    compress
    sharedscripts
    postrotate
        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
    endscript
    missingok
    notifempty
    su root ceph
}

Reply:
ok, if we are not keeping some of these, we should be aware of the reason. Like for below
> sharedscripts
postrotate
killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
endscript
notifempty
su root ceph

Reply:
notifempty : 
do not rotate if log file is empty.
su root ceph :
Rotate log files set under this user and group instead of using default user/group (usually root)
postrotate:
ceph-mon, ceph-mgr etc, these all are daemon services, if we need to reload the services after compression,
killalll is used for that reason.
In our case, we don't want the rgw service to restart after every compress run, so can avoid it.

'notifempty' can be useful

Reply:
ok